src/Action/ConvertPaymentAction.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Marem\PayumPaybox\Action;
+
+use Payum\Core\Action\ActionInterface;
+use Payum\Core\Action\GatewayAwareAction;
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\Model\PaymentInterface;
+use Payum\Core\Request\Convert;
+use Payum\Core\Request\GetCurrency;
+
+class ConvertPaymentAction extends GatewayAwareAction
+{
+    /**
+     * {@inheritDoc}
+     *
+     * @param Convert $request
+     */
+    public function execute($request)
+    {
+        RequestNotSupportedException::assertSupports($this, $request);
+
+        /** @var PaymentInterface $payment */
+        $payment = $request->getSource();
+
+        $token = $request->getToken();
+
+        $dateTime = date("c");
+        $pbx_typeCarte = 'VISA';
+        $pbx_typePaiement = 'CARTE';
+
+        $details = array();
```

Comment:
better to do it like this https://github.com/Payum/Payum/blob/master/src/Payum/Paypal/ExpressCheckout/Nvp/Action/ConvertPaymentAction.php#L29

so a developer be able to pass custom details